zebra-consensus/src/block/subsidy/funding_streams.rs

```diff
@@ -50,3 +48,116 @@ pub fn funding_stream_values(
     }
     Ok(results)
 }
+
+/// Returns the minumum height after the first halving
+/// as described in [protocol specification §7.10][7.10]
+///
+/// [7.10]: https://zips.z.cash/protocol/protocol.pdf#fundingstreams
+fn height_for_halving(network: Network) -> Height {
```

Comment:
Minor: Maybe rename this to make it a bit more distinct from `HeightForHalving` from the spec? Just so that we avoid the ambiguity where they have the same names but only one has a `height` parameter.

```suggestion
fn height_for_first_halving(network: Network) -> Height {
```

Reply:
Good idea, made this at https://github.com/ZcashFoundation/zebra/pull/3040/commits/126f23beca18fd60c28998fd876b293cdb809e93